src/components/Feeds/BusinessFeed.js

```diff
@@ -1,12 +1,21 @@
 import React from 'react';
 import { StaticQuery, graphql } from 'gatsby';
-
+import ResultCard from '../ResultCard';
 import { Box } from '@chakra-ui/core';
 
 const BusinessesFeed = data => {
+  const businesses = data.data.allAirtableBusinesses.nodes;
   return (
     <Box as="pre" whiteSpace="break-spaces">
```

Comment:
Since we're returning the ResultCard, instead of JSON, we can move this `<pre>` wrapper!

Reply:
I removed the `<pre>` it seems like the actual `<Box>` is required to ensure that the width of all of the ResultCards is consistent.